cms/djangoapps/contentstore/views/tests/test_preview.py

```diff
@@ -105,6 +108,37 @@ def test_preview_no_asides(self):
         self.assertNotRegexpMatches(html, r"data-block-type=[\"\']test_aside[\"\']")
         self.assertNotRegexpMatches(html, "Aside rendered")
 
+    @mock.patch('xmodule.conditional_module.ConditionalModule.is_condition_satisfied')
+    def test_preview_conditional_module_children_context(self, mock_is_condition_satisfied):
+        """
+        Testst that when empty context is pass to children of ConditionalModule it will not raise KeyError.
+        """
+        mock_is_condition_satisfied.return_value = True
+        client = Client()
+        client.login(username=self.user.username, password=self.user_password)
+
+        with self.store.default_store(ModuleStoreEnum.Type.split):
+            course_id = self.store.make_course_key('HarvardX', 'ER22x', '2013_Spring')
+            import_course_from_xml(
```

Comment:
why did you have to import the course instead of create it using Course and ItemFactories?